src/collections/_documentation/platforms/javascript/sourcemaps/troubleshooting.md

```diff
@@ -34,21 +40,46 @@ Sentry will look for a matching artifact named exactly `https://example.com/dist
 
 Note also that Sentry will resolve relative paths. For example, if you have the following:
 
-```JavaScript
+```javascript
 // -- end app.min.js (located at https://example.com/dist/js/app.min.js)
 //# sourceMappingURL=app.min.js.map
 ```
 
 Sentry will resolve `sourceMappingURL` relative to `https://example.com/dist/js/` (the root path from which `app.min.js` was served). You will again need to name your source map with the full URL: `https://example.com/dist/js/app.min.js.map`.
```

Comment:
Not your change, but this is confusing: "You will again need to name your source map with the full URL: `https://example.com/dist/js/app.min.js.map`."

We just said you _don't_ have to specify the full path. Which is it? And why _again_?

Reply:
this is actually referring to the uploaded artifact. in bold note that the ~ is a Sentry specific thing